src/sources/tripadvisor/convert.rs

```diff
@@ -82,11 +123,42 @@ pub fn build_poi(
                 .map(ToString::to_string)
         })
         .unwrap_or_else(|| category.clone())
+        .replace(" ", "_")
         .to_lowercase();
 
+    let cuisine = property
+        .cuisine
+        .inner
+        .into_iter()
+        .filter_map(|item| {
+            get_local_string(&["us".to_string()], &build_i18n_property(item.name))
+                .map(ToString::to_string)
+        })
+        .map(|ta_cuisine| {
+            CUISINE_CONVERTER
+                .get(ta_cuisine.to_lowercase().replace(" ", "").as_str())
```

Comment:
Now that the subcategory is using ~snake case, this should probably be changed to be consistent

```suggestion
                .get(ta_cuisine.to_lowercase().replace(" ", "_").as_str())
```